mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_6342 net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider
```

Comment:
What exactly does this mean?

Reply:
I don't know, that's just the name :)

Reply:
I guess the cumulative density function has a trapezoid shape?